tests/config_tests.js

```diff
@@ -246,14 +247,19 @@ describe('Config', function() {
   it('should customize user_data_dir when provided', function(done) {
     var config = new Config();
     stub(config, 'getWantedLaunchers', function(n, cb) {return cb(null, n);});
-    config.set('user_data_dir', 'customDirectory');
+    config.set('user_data_dir', 'node_modules');
     config.getLaunchers(function(err, launchers) {
-      var correctFlag = config.cwd() + '/customDirectory/testem.firefox';
+      var correctFlag = config.cwd() + '/node_modules/testem.firefox';
       if (process.platform === 'win32') {
-        correctFlag = config.cwd() + '\\customDirectory\\testem.firefox';
+        correctFlag = config.cwd() + '\\node_modules\\testem.firefox';
       }
       expect(launchers.firefox.settings.args).to.include(correctFlag);
-      done();
+      launchers.firefox.settings.setup(config, function() {
+        fs.exists(correctFlag, function(exists) {
```

Comment:
Hm, `fs.exists` is deprecated https://nodejs.org/api/fs.html#fs_fs_exists_path_callback, could you use https://nodejs.org/api/fs.html#fs_fs_stat_path_callback instead?